lib/animate_icons.dart

```diff
@@ -118,8 +122,8 @@ class _AnimateIconsState extends State<AnimateIcons>
   Widget build(BuildContext context) {
     double x = _controller.value;
     double y = 1.0 - _controller.value;
-    double angleX = math.pi / 180 * (180 * x);
-    double angleY = math.pi / 180 * (180 * y);
+    double angleX = math.pi / 180 * ((widget.amplitude ?? 180) * x);
+    double angleY = math.pi / 180 * ((widget.amplitude ?? 180) * y);
```

Comment:
and remove the default value from here
`double angleY = math.pi /180 * (widget.amplitude * y);`